root/components/forms.tt

```diff
@@ -352,7 +352,12 @@
       [% FOREACH id IN form.options %]
          [%- id = id.value -%]
          <span [% IF form.value != id %] style="display:none" [% END %] class="type-bubble-description" id="type-bubble-description-[% id %]">
-           <strong>[%- add_colon(l('Description')) -%] </strong>[%- types.$id.l_description -%]
+           <strong>[%- add_colon(l('Description')) -%] </strong>
```

Comment:
I know this is already being displayed for work type but it seems neither needed nor helpful.

Reply:
Reported in MBS-11177.